crates/motoko_proc_macro/src/lib.rs

```diff
@@ -10,22 +11,10 @@ pub fn parse_static(stream: TokenStream) -> TokenStream {
     let input = syn::parse::<syn::LitStr>(stream)
         .expect("Unexpected macro input")
         .value();
-
     let prog = motoko::check::parse(&input).expect("Evaluation error");
     let serialized = motoko::proc_macro::serialize(&prog).expect("Serialization error");
-    // let value: motoko::value::Value = bincode::deserialize(&serialized).unwrap();
-
     let serialized_ast = proc_macro2::Literal::byte_string(&serialized);
-
-    let _ = motoko::proc_macro::deserialize::<motoko::ast::Prog>(&serialized)
-        // .map_err(|e| e.to_string())
-        .unwrap();
-
     TokenStream::from(quote! {{
-        // lazy_static! {
-        //     static ref VALUE: Value = deserialize_from(&#serialized_tokens).unwrap();
-        // }
-        // VALUE
-        serde_json::deserialize::<motoko::ast::Prog>(&#serialized_ast)/* .map_err(|e| e.to_string()) */.unwrap()
+        motoko::proc_macro::deserialize::<motoko::ast::Prog>(#serialized_ast as &[u8]).unwrap()
```

Comment:
Here's the place where `serde_json` was still being mentioned.  I used the new name that you created and overcame that issue.

Reply:
One remaining item for polish is the use of `lazy_static` -- I found the errors from misusing that to be very confusing, and elided it to get things to work.

Perhaps you can take a look, or we can look together again soon.